editor/scene_tree_editor.cpp

```diff
@@ -1351,6 +1422,7 @@ SceneTreeEditor::~SceneTreeEditor() {
 
 void SceneTreeDialog::popup_scenetree_dialog() {
 	popup_centered_clamped(Size2(350, 700) * EDSCALE);
+	//set_title("Potato");
```

Comment:
Duh :)

Reply:
Oops. Old leftover, while attempting to solve another issue.